src/test/java/hw6/BaseClassTest.java

```diff
@@ -0,0 +1,45 @@
+package hw6;
+
+import hw6.service.webdriver.WebDriverSingleton;
+import hw6.steps.ActionStep;
+import hw6.steps.AssertionStep;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class BaseClassTest {
+    protected WebDriver webDriver;
+    protected Configuration properties;
+    private final String pathToProperties = "src/test/resources/hw3Properties/pageData.properties";
+
+    protected ActionStep actionStep;
+    protected AssertionStep assertionStep;
+
+    @BeforeClass
+    public void setUp(ITestContext testContext) throws ConfigurationException {
+        webDriver = WebDriverSingleton.getWebDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+        properties = new PropertiesConfiguration(pathToProperties);
```

Comment:
Fails here with error
```java
[ERROR] Failures: 
[ERROR]   HomePageTest>BaseClassTest.setUp:31 » Configuration Cannot locate configuratio...
[ERROR]   DifferentElementsTest>BaseClassTest.setUp:31 » Configuration Cannot locate con...
```
Cannot find configuration file
Please refactor

Reply:
added to repository